examples/socketio/README.md

```diff
@@ -0,0 +1,81 @@
+# socketio
+
```

Comment:
could you please a one-liner description of this example? It will be used in the example list page as well.